src/components/dialogs/network-modifications/generator/reactive-limits/reactive-limits-form.js

```diff
@@ -85,13 +84,6 @@ const ReactiveLimitsForm = ({
 
     return (
         <>
```

Comment:
Suggestion: remove  **<> </>** no need for it.

Reply:
done